csharp/src/SeedLang.Block/IEditable.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Aha001 Team.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace SeedLang.Block {
+  // All editable blocks should implement this interface to provide the content getter and setter to
+  // the UI layer.
+  public interface IEditable {
+    // Gets the editable block content, a plain text code.
```

Comment:
Don't understand the meaning of "code". I guess it's the content of the editable block?

Reply:
I intentionally implied the meaning of source code, since the blocks of SeedBlock are comparable to source code elements. I understand that this metaphor is a bit confusing here. Let's back off to the clearest term "text" - every editable block has an inner text to be edited.